src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -188,11 +191,13 @@ def test_edit_as_developer(self):
 
         assert unicode(addon.name) == data['name']
 
-        assert unicode(addon.slug) == data['slug']
         assert unicode(addon.summary) == data['summary']
 
         if self.listed:
             assert [unicode(t) for t in addon.tags.all()] == sorted(self.tags)
+            # We randomize slugs for unlisted add-ons so we can't rely
+            # on the output
+            assert unicode(addon.slug) == data['slug']
```

Comment:
little confused here too - we don't randomize the slug on each save do we?

Reply:
No, we don't

Reply:
But we just set the slug so it shouldn't be randomized though?  Can we have tests confirming that you can a) change the slug from a randomized one and b) it doesn't change from one randomized slug to another randomized slug on save.